mocks/react-mocks/src/components/NavBar/index.jsx

```diff
@@ -7,25 +8,25 @@ export default function NavBar() {
   return (
     <React.Fragment>
       <div className={styles.skipLink}>
-        <a href="#mainContent">Skip to Main Content</a>
+        <Link to="#mainContent">Skip to Main Content</Link>
```

Comment:
Feel like this should remain as:
```html
<a href="#mainContent">Skip to Main Content</a>
```
Think this breaks skip link used for accessibility. Thoughts?

Reply:
Links do end up rendering an anchor tag in the DOM, but good point - i'm not sure how it treats skip links so i'll revert it.

Reply:
Yeah its more so hashed links in general. Reverting fixed it! Will do another quick glance but everything seems to look good!